common/src/main/webapp/sass/explore.scss

```diff
@@ -157,7 +161,212 @@ $tree-section-width: 300px;
       display: unset;
     }
   }
+}
+
+// ------
+// Sidereal Target Editor
+// ------
+.explore-brightnesses-wrapper {
+  grid-column: 1 / -1;
+  justify-self: stretch;
+  align-self: stretch;
+}
+
+.explore-brightnesses-container {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.explore-brightnesses-units-dropdown.p-dropdown {
+  white-space: nowrap;
+  width: 100%;
+}
+
+.explore-brightnesses-delete-button-wrapper {
+  display: flex;
+  justify-content: center;
+
+  .p-button.delete-button {
+    padding: 0;
+    flex-shrink: 0;
+  }
+}
+
+.target-grid {
+  display: grid;
+  grid-gap: 0.5em;
+  grid-row-gap: 0.3em;
+  padding: 0.5em;
+  height: 100%;
+
+  .explore-title-undo-buttons {
+    grid-area: undo;
+  }
+
+  .target-aladin-cell {
+    grid-area: aladin;
+  }
+
+  .target-form {
+    grid-area: coordinates;
+  }
+
+  .target-proper-motion-form {
+    grid-area: proper-motion;
+
+    .p-inputgroup>span:first-of-type {
+      width: 4em;
+    }
+  }
+
+  .target-source-profile-editor {
+    grid-area: source-profile;
+  }
+}
+
+.target-rv-controls {
+  justify-self: stretch;
+
+  .explore-grow-1 {
+    margin-left: 0.3em !important;
+  }
+}
+
+@mixin expanded-aladin {
+  grid-template-areas: 'aladin';
+  grid-template-columns: 1fr;
+
+  .explore-title-undo-buttons {
+    display: none;
+  }
 
+  .target-form {
+    display: none;
+  }
+
+  .target-proper-motion-form {
+    display: none;
+  }
+
+  .target-source-profile-editor {
+    display: none;
+  }
+}
+
+.tile-xl-width,
+.tile-lg-width,
+.tile-md-width,
+.tile-sm-width {
+  .target-grid {
+    grid-template-columns: 1fr 2fr;
+    grid-template-rows: min-content min-content min-content 1fr;
+    grid-template-areas:
+      'undo           aladin'
+      'coordinates    aladin'
+      'proper-motion  aladin'
+      'source-profile aladin';
+  }
+
+  .target-aladin-cell {
+    grid-row: span 4;
+  }
+}
+
+.tile-xs-width {
+  
+  // undo buttons are not show in xs tiles, only aladin
```

Comment:
Will delete this and the stuff below before merging if we are going to keep the current layouts mentioned above.